subsys/bluetooth/audio/vocs_client.c

```diff
@@ -634,6 +634,28 @@ struct bt_vocs *bt_vocs_client_free_instance_get(void)
 	return NULL;
 }
 
+int bt_vocs_client_conn_get(struct bt_vocs *vocs, struct bt_conn **conn)
```

Comment:
vocs is not changed in this function. Suggest `struct bt_vocs const * const vocs`